sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastWithAnsiOffSuite.scala

```diff
@@ -608,6 +608,11 @@ class CastWithAnsiOffSuite extends CastSuiteBase {
     checkEvaluation(cast(input, StringType), "1.23E-7")
   }
 
+  test("SPARK-42176: cast boolean to timestamp") {
```

Comment:
The PR title seems to show a wrong JIRA ID, `SPARK-42175`. Let me fix it.

Reply:
Oh, thanks for fixing it!